Store/BLL/Interfaces/IProductBLL.cs

```diff
@@ -0,0 +1,13 @@
+﻿using BLL.Models;
+
+namespace BLL.Interfaces
+{
+    public interface IProductBLL
```

Comment:
to rename on something smooth/clean/logic

Reply:
Better to name it like IProductService